LocalMultiplayerAgent.UnitTest/DeploymentSettingsTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using FluentAssertions;
+using Microsoft.Azure.Gaming.LocalMultiplayerAgent.DeploymentTool;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Microsoft.Azure.Gaming.LocalMultiplayerAgent.UnitTests
+{
+    [TestClass]
+    public class DeploymentSettingsTests
```

Comment:
this class tests only for "false" outcomes, we should make sure that "true" outcomes work as well.